sdk/iot/inc/az_iot_sas_token.h

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef AZ_IOT_SAS_TOKEN_H
+#define AZ_IOT_SAS_TOKEN_H
+
+#include <stdlib.h>
+#include <az_result.h>
+#include <az_span.h>
+
+#include <_az_cfg_prefix.h>
+
+typedef struct az_iot_sas_params {
```

Comment:
Should these all be public? My guess is NO

Reply:
Ooops, this was not meant to be sent on the PR. We are avoiding using structs for params, as per insights shared by @cipop.

Reply:
Removed.